### PR TITLE
add phone_numbers endpoint

### PR DESCRIPTION
**Pull request by lunarfyre7**

### CODE REVIEW

File: lib/telnyx/phone_number.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Telnyx
+  class PhoneNumber < APIResource
+    extend Telnyx::APIOperations::Create
```
Comment (eoghantelnyx) [inline lib/telnyx/phone_number.rb]: I'm not sure what this is for?

File: lib/telnyx/phone_number.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    extend Telnyx::APIOperations::NestedResource
+    include Telnyx::APIOperations::Save
+
+    ACTIONS = %w[voice messaging].freeze
+
+    ACTIONS.each do |action|
+      nested_resource_class_methods action,
+                                    path: [action],
+                                    operations: %i[create list],
```
Comment (eoghantelnyx) [inline lib/telnyx/phone_number.rb]: List isn't working for me

Comment (lunarfyre7) [reply lib/telnyx/phone_number.rb]: Alright, things should work right now

**Comment by lunarfyre7** [comment]
Alright, it should work correctly now.

**Pull request merged**